packages/google_maps_flutter/lib/src/google_maps_controller.dart

```diff
@@ -173,10 +173,10 @@ class GoogleMapsController {
     final String options = _createOptions();
     final String command = '''
       map = new google.maps.Map(document.getElementById('map'), $options);
-      map.addListener('bounds_changed', BoundChanged.postMessage);
-      map.addListener('idle', Idle.postMessage);
-      map.addListener('click', (event) => Click.postMessage(JSON.stringify(event)));
-      map.addListener('tilesloaded', Tilesloaded.postMessage);
+      map.addListener('bounds_changed', (event) => { BoundChanged.postMessage('') });
+      map.addListener('idle', (event) => { Idle.postMessage('') });
+      map.addListener('click', (event) => { Click.postMessage(JSON.stringify(event)) });
+      map.addListener('tilesloaded', (evnet) => { Tilesloaded.postMessage('') });
```

Comment:
How about add semicolons
```suggestion
      map.addListener('bounds_changed', (event) => { BoundChanged.postMessage(''); });
      map.addListener('idle', (event) => { Idle.postMessage(''); });
      map.addListener('click', (event) => { Click.postMessage(JSON.stringify(event)); });
      map.addListener('tilesloaded', (evnet) => { Tilesloaded.postMessage(''); });
```

Reply:
Done.

Reply:
oh my comment about CHANGELOG is missing. Could you please update CHANGELOG.md as well?
```
## NEXT

* Update webview_flutter_lwe to 0.3.3.
* Fix memory alarm and map initialization issue
```

Reply:
Sure, I have updated it.